src/lib/utils/DY.Lib.Printing.fst

```diff
@@ -87,12 +87,12 @@ val usage_to_string: (u:usage) -> string
 let usage_to_string u =
   match u with
   | NoUsage -> "NoUsage"
-  | SigKey tag -> "SigKey " ^ tag
+  | SigKey tag _ -> "SigKey " ^ tag
   | SigNonce -> "SigNonce"
-  | PkdecKey tag -> "PkdecKey " ^ tag
+  | PkdecKey tag _ -> "PkdecKey " ^ tag
   | PkNonce -> "PkNonce"
-  | AeadKey tag -> "AeadKey " ^ tag
-  | DhKey tag -> "DhKey " ^ tag
+  | AeadKey tag data -> Printf.sprintf "AeadKey %s (data=(%s))" tag (bytes_to_string data)
```

Comment:
Should the other data-containing usages get the same treatment?

Reply:
The question would be whether we want to print the data at all? This will most likely blow up the output quite a lot.

Reply:
As you wish @fabian-hk, I guess it can make sense for AeadKey and KdfExpandKey because I expect that the `data` field can often be useful with these keys, but we may also always omit them.

Reply:
I implemented a version in which the usage is printed as a JSON object: #43. This gives the full output on the command line but still allows us to automatically filter the output with a third-party script.